context/app/main.py

```diff
@@ -54,15 +56,15 @@ def create_app(testing=False):
     app.register_error_handler(404, not_found)
     app.register_error_handler(504, gateway_timeout)
 
-    @app.context_processor
+    @ app.context_processor
```

Comment:
```suggestion
    @app.context_processor
```
I think no space is more usual?